devicetypes/sinope-technologies/va4200wz-va4200zb-sinope-valve.src/va4200wz-va4200zb-sinope-valve.groovy

```diff
@@ -11,9 +11,10 @@ import physicalgraph.zigbee.zcl.DataType
 metadata {
 
 	preferences {
-		input("trace", "bool", title: "Trace", description: "Set it to true to enable tracing")
-		// input("logFilter", "number", title: "Trace level", range: "1..5",
-		// 	description: "1= ERROR only, 2= <1+WARNING>, 3= <2+INFO>, 4= <3+DEBUG>, 5= <4+TRACE>")
+        input("trace", "bool", title: "Trace (Only for debugging)", description: "Set it to true to enable tracing")
+		
+		//input("logFilter", "number", title: "Trace level", range: "1..5",
+		//description: "1= ERROR only, 2= <1+WARNING>, 3= <2+INFO>, 4= <3+DEBUG>, 5= <4+TRACE>")
```

Comment:
nit: spacing (please use tabs) and indentations
additionally: do You still need to keep those lines that were commented out?

Reply:
In my ide, it's all properly indented. I don't know what's going on. 

I'll remove the commented code.